### PR TITLE
Fix the crash caused by the key press

### PR DESCRIPTION
**Pull request by vSylva**
https://github.com/veeenu/hudhook/issues/172

### CODE REVIEW

File: src/renderer/input.rs [hunk 1]
```diff
@@ -55,26 +55,44 @@ fn handle_raw_mouse_input(io: &mut Io, raw_mouse: &RAWMOUSE) {
...
+    // event(RI_MOUSE_LEFT_BUTTON_DOWN, MouseButton::Left, VK_LBUTTON, true);
+    // event(RI_MOUSE_LEFT_BUTTON_UP, MouseButton::Left, VK_LBUTTON, false);
+    // event(RI_MOUSE_RIGHT_BUTTON_DOWN, MouseButton::Right, VK_RBUTTON, true);
+    // event(RI_MOUSE_RIGHT_BUTTON_UP, MouseButton::Right, VK_RBUTTON, false);
+    // event(RI_MOUSE_MIDDLE_BUTTON_DOWN, MouseButton::Middle, VK_MBUTTON, true);
+    // event(RI_MOUSE_MIDDLE_BUTTON_UP, MouseButton::Middle, VK_MBUTTON, false);
+    // event(RI_MOUSE_BUTTON_4_DOWN, MouseButton::Extra1, VK_XBUTTON1, true);
+    // event(RI_MOUSE_BUTTON_4_UP, MouseButton::Extra1, VK_XBUTTON1, false);
+    // event(RI_MOUSE_BUTTON_5_DOWN, MouseButton::Extra2, VK_XBUTTON2, true);
+    // event(RI_MOUSE_BUTTON_5_UP, MouseButton::Extra2, VK_XBUTTON2, false);
```
Comment (veeenu) [inline src/renderer/input.rs:95]: Could you please remove these?

**Review by veeenu — CHANGES_REQUESTED** [review]
Save from the commented lines, looks good to me!

**Review by vSylva — COMMENTED** [review]
Remove Comment https://github.com/veeenu/hudhook/pull/175#discussion_r1523791496

File: src/renderer/input.rs (hunk 1 above)
Comment (vSylva) [reply src/renderer/input.rs]: okie
done

**Comment by veeenu** [comment]
Thank you!

**Pull request merged**